internal/proxyservice/proxy_service.go

```diff
@@ -1,43 +1,147 @@
 package proxyservice
 
 import (
+	"net"
+	"strings"
+
+	"github.com/cyberark/secretless-broker/internal"
 	"github.com/cyberark/secretless-broker/internal/plugin"
 	v1 "github.com/cyberark/secretless-broker/internal/plugin/v1"
+	tcpproxy "github.com/cyberark/secretless-broker/internal/proxyservice/tcp"
 	"github.com/cyberark/secretless-broker/pkg/secretless"
 	v2 "github.com/cyberark/secretless-broker/pkg/secretless/config/v2"
-	"github.com/cyberark/secretless-broker/pkg/secretless/log"
+	logapi "github.com/cyberark/secretless-broker/pkg/secretless/log"
+	"github.com/cyberark/secretless-broker/pkg/secretless/plugin/connector"
+	"github.com/cyberark/secretless-broker/pkg/secretless/plugin/connector/tcp"
 )
 
 // TODO: move to impl package
 type proxyServices struct {
-	config        v2.Config
-	logger        log.Logger
-	eventNotifier v1.EventNotifier
-	availPlugins  plugin.AvailablePlugins
+	config          v2.Config
+	logger          logapi.Logger
+	eventNotifier   v1.EventNotifier
+	availPlugins    plugin.AvailablePlugins
+	runningServices []secretless.Service
 }
 
-// TODO: Rename to Call or Run and return a Stopper instead of having Stop()
-// Start starts all proxy services
-func (s *proxyServices) Start() {
-	// TODO: Implement
+type connectorResources struct {
+	logger logapi.Logger
+	config []byte
+}
```

Comment:
Code duplication. Why not just make [this](https://github.com/cyberark/secretless-broker/blob/master/pkg/secretless/plugin/connector/resources.go#L19-L30) a public struct instead?